TrkDiag/test/TOTDriftTime.py

```diff
@@ -0,0 +1,71 @@
+import ROOT
```

Comment:
I think it's great you wrote this script.  For this PR it's fine to keep it in Offline, longterm I think we want to move this and other similar analysis outside of Offline, maybe into a TrackerCalib repo.